bin/tests/test_model/titanic_model.py

```diff
@@ -23,10 +23,13 @@ def forward(self, pclass: torch.tensor, sex: torch.tensor, age: torch.tesnor, si
     def compute_loss(self, loss_fn, output, survived):
         return loss_fn(output, survived)
     
-    def batch(self, x: dict, y: dict, loss_fn: Callable, optimizer: Callable):
-
+    def batch(self, x: dict, y: dict, loss_fn: Callable, optimizer: Callable = None):
         output = self.forward(**x)
         loss = self.compute_loss(loss_fn, output, **y)
+
+        if optimizer is None:
```

Comment:
do we want to allow this? 

When do we have optimizer None? maybe we should instead have a default?

Reply:
okay, i agree, i will change this